This resolves a handful of CVEs:

* CVE-2023-36479
* CVE-2023-40167
* CVE-2023-41900
* GHSA-hmr7-m48g-48f6

---
diff --git a/pom.xml b/pom.xml
index 591d9cd..c3a35af 100644
--- a/pom.xml
+++ b/pom.xml
@@ -99,7 +99,7 @@
     <dependency>
       <groupId>org.eclipse.jetty</groupId>
       <artifactId>jetty-servlet</artifactId>
-      <version>11.0.15</version>
+      <version>11.0.16</version>
     </dependency>
     <dependency>
       <groupId>com.github.ben-manes.caffeine</groupId>
